### PR TITLE
Ccxt connector refactoring (multi subscription)

### PR DESCRIPTION
**Pull request by yuriy-arabskyy**

### CODE REVIEW

File: src/qubx/core/interfaces.py [hunk 1]
```diff
@@ -403,7 +417,7 @@ def cancel_order(self, order_id: str) -> None:
 class IUniverseManager:
     """Manages universe updates."""
 
-    def set_universe(self, instruments: list[Instrument]):
+    def set_universe(self, instruments: list[Instrument], skip_callback: bool = False):
         """Set the trading universe.
```
Comment (dmarienko) [inline src/qubx/core/interfaces.py:421]: Q: do we really need skip_callback parameter ?

File: src/qubx/core/mixins/universe.py [hunk 2]
```diff
@@ -58,7 +58,7 @@ def __init__(
         self.__position_gathering = position_gathering
         self._instruments = []
 
-    def set_universe(self, instruments: list[Instrument]) -> None:
+    def set_universe(self, instruments: list[Instrument], skip_callback: bool = False) -> None:
```
Comment (dmarienko) [inline src/qubx/core/mixins/universe.py:61]: Do we really need skip_callback argument ?

File: src/qubx/core/mixins/processing.py
Comment (dmarienko) [inline src/qubx/core/mixins/processing.py:1]: Do you think we could return any trading signals from IStrategy.on_market_data method ?
Or you need that for features framework ?

File: src/qubx/core/interfaces.py (hunk 1 above)
Comment (yuriy-arabskyy) [reply src/qubx/core/interfaces.py:405]: I agree, it's something that I don't like as well. Added this temporarily, will think how to improve. I was thinking that I don't want the initial instruments to go through the on_universe and I want them to be available in the on_start method. And this parameter was related to this choice. But there is likely some more elegant solution.

File: src/qubx/core/mixins/processing.py
Comment (yuriy-arabskyy) [reply src/qubx/core/mixins/processing.py:1]: you mean not to return any signals right? I don't really need that, just wanted to leave this possibility open. But maybe it is unnecessary. What is your opinion?

**Pull request merged**